interop-tests/Dockerfile.chromium

```diff
@@ -14,13 +14,13 @@ FROM chef AS builder
 COPY --from=planner /app/recipe.json recipe.json
 # Build dependencies - this is the caching Docker layer!
 RUN cargo chef cook --release --package interop-tests --target wasm32-unknown-unknown --recipe-path recipe.json
-RUN cargo chef cook --release --package interop-tests --bin wasm_ping --recipe-path recipe.json
+RUN RUSTFLAGS='-C target-feature=+crt-static' cargo chef cook --release --package interop-tests --target x86_64-unknown-linux-gnu --bin wasm_ping --recipe-path recipe.json
```

Comment:
Can you add a short comment why this is needed? For future us to eventually go back to the provided glibc?